src/main/java/org/fcrepo/migration/validator/impl/Fedora3ObjectValidator.java

```diff
@@ -64,6 +64,7 @@ public Fedora3ObjectValidator(final OcflObjectSessionFactory factory,
 
         try {
             object.processObject(new ObjectAbstractionStreamingFedoraObjectHandler(handler));
+            object.close();
```

Comment:
Does this not need closed on exception too?

Reply:
Saw that the FedoraObjectProcessor extends AutoCloseable so I updated this to use try-with-resources